### PR TITLE
Conductor-Managed BYOL (AWS)

### PR DESCRIPTION
**Pull request by haberkornsam**

### CODE REVIEW

**Review by MichaelBaj — DISMISSED** [review]

**Comment by MichaelBaj** [comment]
These links appear to no longer be valid. Please replace with current equivalent or rework documentation.
```
ERROR: 2 dead links found!
[✖] https://docs.saltstack.com/en/latest/ref/states/top.html → Status: 404
[✖] https://docs.saltstack.com/en/latest/topics/tutorials/pillar.html → Status: 404
```

File: docs/intro_installation_quickstart_byol_mist_aws.md [hunk 1]
```diff
@@ -305,11 +296,7 @@ To use the template for the **Bring Your Own License (BYOL)** image was used, SS
     If the desired key pair is not listed click **Create a new key pair**, enter a name in the Key pair name field and click **Download Key Pair**.
     :::
 21. Check the acknowledgment check box and then click **Launch Instances**.
-22. On the _Launch Status_ page, click **View Instances**.
-23. Record the instances IP address.
-24. Launch a command window prompt.
-25. Enter the IP address of the instance.
-26. When prompted by the installer, press the **Enter** key to select Begin.
+22. If an onboarding configuration was not provided in step 16, follow the manual steps to in the [Manual Onboarding](#manual-onboarding) section.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_installation_quickstart_byol_mist_aws.md:299]:
```suggestion
22. If an onboarding configuration was not provided in step 16, follow the steps in the [Manual Onboarding](#manual-onboarding) section.
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]
Sam, I had several edits in the intro_installation_quickstart_byol_conductor_aws.md topic - most were purely editorial. However I added comments in the doc itself for clarification of two steps in two procedures - steps 16 and 22. They call out mist managed devices, and then reference config templates. Please take a look at them and if you have questions, let me know. Thanks.

File: docs/intro_installation_quickstart_byol_conductor_aws.md [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+18. In the _Boot from General Purpose_ window, select **Continue** and click **Next**.
+19. On the _Step 7: Review Instance Launch_ page, click **Launch** to finalize the instance.
+20. In the _Select an existing key pair or create a new key pair_ dialog box, select **Choose an existing key pair** and select the desired key pair from the list.
+    :::note
+    If the desired key pair is not listed click **Create a new key pair**, enter a name in the Key pair name field and click **Download Key Pair**.
+    :::
+21. Place a check the acknowledgment check box and  click **Launch Instances**.
+22. If an onboarding configuration was not provided in step 16, follow the steps in the [Manual Onboarding](#manual-onboarding) section.
+
+## Session Smart Router Deployment
```
Comment (agrawalkaushik) [inline docs/intro_installation_quickstart_byol_conductor_aws.md:259]: can we call this `Session Smart Conductor Managed Router` isntead?

**Review by agrawalkaushik — APPROVED** [review]

File: docs/release_notes_byol_2.0.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+## Release 2.0.0
+
+**Release Date:** October 24, 2024
+
+### New Features
+
+- **I95-57252 BYOL Support for Conductor and Conductor Managed router:** Added support for IBU Conductors. BYOL Conductors are supported with 6.3.0-R1 and above. Added support for IBU Conductor Managed routers. Conductor managed routers require 6.1.0 or above and must be managed by a 6.3.0-R1 or greater conductor.
+
+- **I95-58706 Whitelist AWS cXa instance types:** Added support for AWS cXa instance types such as c7a. These instnace types utilize varius AMD processors. More information can be found in the [AWS documentation](https://aws.amazon.com/ec2/instance-types/#Compute_Optimized).
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_byol_2.0.md:13]:
```suggestion
- **I95-58706 Whitelist AWS cXa instance types:** Added support for AWS cXa instance types such as c7a. These instance types utilize various AMD processors. For more information, please see the [AWS documentation](https://aws.amazon.com/ec2/instance-types/#Compute_Optimized).
```

File: docs/release_notes_byol_2.0.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+### Resolved Issues
+
+- **I95-58549 Azure and AWS instances can sometimes fail to successfully connect to Mist:** On first boot the system can sometimes fail to connect to Mist before timing out. This causes the SSR software to not be installed. The timeout is now increased.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/release_notes_byol_2.0.md:17]:
```suggestion
- **I95-58549 Azure and AWS instances can sometimes fail to successfully connect to Mist:** On first boot the system can sometimes fail to connect to Mist before timing out. This causes the SSR software to not be installed. The timeout has been increased.
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]
I have to check a couple things in the intro_installation topic as well, because I'm not seeing changes I made directly to the doc.

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Pull request merged**